src/libfetchers/git.cc

```diff
@@ -147,9 +147,10 @@ std::vector<PublicKey> getPublicKeys(const Attrs & attrs)
 {
     std::vector<PublicKey> publicKeys;
     if (attrs.contains("publicKeys")) {
-        nlohmann::json publicKeysJson = nlohmann::json::parse(getStrAttr(attrs, "publicKeys"));
-        ensureType(publicKeysJson, nlohmann::json::value_t::array);
-        publicKeys = publicKeysJson.get<std::vector<PublicKey>>();
+        nlohmann::json publicKeysJson = getArray(nlohmann::json::parse(getStrAttr(attrs, "publicKeys")));
+        for (auto & jsonKey : publicKeysJson) {
```

Comment:
```suggestion
        for (auto & jsonKey : getArray(nlohmann::json::parse(getStrAttr(attrs, "publicKeys")))) {
```
Can just combine everything again, I think!